waku/node/peer_manager/peer_manager.nim

```diff
@@ -665,8 +665,11 @@ proc connectToNodes*(
   when defined(debugDiscv5):
     let peerIds = connectedPeers.mapIt(it.peerId)
     let origin = connectedPeers.mapIt(it.origin)
-    notice "established connections with found peers",
-      peerIds = peerIds, origin = origin
+    if peerIds.len > 0:
+      notice "established connections with found peers",
+        peerIds = peerIds, origin = origin
```

Comment:
Shall we have a shorter log line?
```suggestion
        peerIds = peers = toSeq(peerIds.mapIt(shortLog(it))), origin = origin
```

Reply:
I'm down for a shorter log line, but have to check if this would make the life harder for DST team or not. @AlbertoSoutullo WDYT? Is it the same for you if we log the whole peerId or just the compressed version?

Reply:
The compressed version of the peerId is also fine c:
It doesn't really make a difference